clientd3d/graphics.c

```diff
@@ -202,7 +202,7 @@ void GraphicsAreaResize(int xsize, int ysize)
    main_viewport_width = view.cx;
    main_viewport_height = view.cy;
    player_overlay_scaler = (float)(main_viewport_width - CLASSIC_WIDTH) / CLASSIC_WIDTH;
-   player_overlay_scaler = (player_overlay_scaler > 0) ? 1.0f + player_overlay_scaler : 1.0f - player_overlay_scaler;
+   player_overlay_scaler = (player_overlay_scaler > 0) ? 1.0f + player_overlay_scaler : 1.0f + player_overlay_scaler;
```

Comment:
Both branches yield the same value, so it looks like these two lines can be simplified, maybe to
```
player_overlay_scaler = ((float) main_viewport_width) / CLASSIC_WIDTH;
```